src/js/index.js

```diff
@@ -1,7 +1,38 @@
 const onUserCreateHandler = () => {
   const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+
+  if(userName.length < 2) {
+    alert("유저 이름은 2글자 이상이어야 합니다.");
+    return;
+  }
+
+  const userList = document.querySelectorAll('#user-list button');
+  for (let user of userList) {
+    if(user.classList.contains('active')) user.classList.remove('active');
+  }
+  
+  const newUser = document.createElement('button');
+  newUser.textContent = userName;
+  newUser.classList.add('ripple','active');
+  userList[0].before(newUser);  
+}
+
+const onUserDeleteHandler = () => {
+  const userList = document.querySelectorAll('#user-list button');
+  for (let user of userList) {
+    if(user.classList.contains('active')) {
+      if(confirm(user.innerHTML + "을(를) 삭제하시겠습니까?")) {
+        user.parentNode.removeChild(user);
+        userList[1].classList.add('active');
+        return;
+      }
+    }
```

Comment:
if문을 중첩해서 사용하면 가독성이 떨어지는것 같아요
active class를 가지고 있지않은 것들은 continue 로 넘어가도록 예외를 두면 
if문을 중첩하지 않고 사용할 수 있어서 조금 좋을 것 같다는 생각이 듭니다.
```suggestion
    if(!user.classList.contains('active')) continue
    
    if(confirm(user.innerHTML + "을(를) 삭제하시겠습니까?")) {
      user.parentNode.removeChild(user);
      userList[1].classList.add('active');
      return;
    }
```

Reply:
좋은 조언과 예시 감사합니다 :-) 리팩토링 할 때 꼭 참고해서 반영하겠습니다!